tests/aws_iot_client_test.c

```diff
@@ -42,9 +43,8 @@
 #    include <unistd.h>
 #endif
 
-AWS_STATIC_STRING_FROM_LITERAL(s_client_id, "aws_iot_client_test");
-AWS_STATIC_STRING_FROM_LITERAL(s_subscribe_topic, "a/b");
-AWS_STATIC_STRING_FROM_LITERAL(s_hostname, "a1ba5f1mpna9k5-ats.iot.us-east-1.amazonaws.com");
+AWS_STATIC_STRING_FROM_LITERAL(s_client_id, "aws_c_mqtt_test_client");
```

Comment:
You're going to want to make sure this has some sort of relatively unique ID attached to it when you connect, otherwise multiple hosts running the tests simultaneously will disconnect each other. It has happened to me.

Reply:
Dang you right, will sprintf something